common/lib/xmodule/xmodule/lti_module.py

```diff
@@ -558,6 +557,12 @@ def role(self):
         }
         return roles.get(self.system.get_user_role(), u'Student')
 
+    @property
+    def icon_class(self):
+        if self.has_score:
```

Comment:
Score, not graded? A scored but not graded problem could be like a sample problem shown to the user. Or like a multiple-choice poll of learners. Might not want the `problem` icon in that case.

Reply:
I'm checking if these problems have access to a graded attribute. Based on the file it was defined in and the fields defined as part of LTI, it seemed to only have `has_score` and not graded, but I agree, graded would be preferred.

Reply:
Ah okay, I forgot that graded is set at a different level (you can't set it on the xblock, but you set the whole subsection as graded and that is passed down to all children). Okay, yes, I will check for `graded` and `has_score` in that case.